### PR TITLE
#320 - Add Summary to Project type & page

### PR DESCRIPTION
**Pull request by RChandler234**
We added summary to the Project page below the project details as a PageBlock and changed the Project type to include the summary. We also updated the dummy data to include dummy summary data.

### CODE REVIEW

**Comment by acd124** [comment]
The p tag could probably be changed to an empty element, whether this is done or not is up to whether the format of the p tag is wanted or not and the preference for style.

File: src/components/projects/wbs-details/project-container/project-container.tsx [hunk 1]
```diff
@@ -27,11 +27,12 @@ const ProjectContainer: React.FC<ProjectContainerProps> = ({ wbsNum }: ProjectCo
   if (isLoading) return <LoadingIndicator />;
 
   if (isError) return <ErrorPage message={error?.message} />;
-
+  console.log(data!.summary);
```
Comment (jamescd18) [inline src/components/projects/wbs-details/project-container/project-container.tsx:30]: Be sure to not leave extraneous `console.log` statements in the code.

**Review by jamescd18 — CHANGES_REQUESTED** [review]
This looks great! One small edit and it should be ready to go!

Few notes:
- be sure to attach a screenshot of the updated UI since your ticket required changing the UI
- be sure to add me/kincent to the PR for review in addition to ping'ing on slack

**Comment by jamescd18** [comment]
Also be sure to link the associated issue on the right!

**Comment by jpwaves** [comment]
> Also be sure to link the associated issue on the right!

Just added issue #320 to "Linked Issues" category on the right side-bar.

**Comment by acd124** [comment]
With p tag 
Screen Shot 2021-09-21 at 12 25 33 PM
Without p tag 
Screen Shot 2021-09-21 at 12 25 49 PM

**Review by jamescd18 — APPROVED** [review]
Ah yes, without the p tag is much better since it eliminates the unnecessary padding on the bottom and the text looks no different. Nice job guys!

**Pull request merged**